src/script/interpreter.cpp

```diff
@@ -1937,6 +1988,56 @@ bool EvalScript(std::vector<std::vector<unsigned char> >& stack, const CScript&
                 }
                 break;
 
+                case OP_ECMULSCALAREXPVERIFY:
+                {
+                    // OP_ECMULSCALAREXPVERIFY is available post tapscript
+                    if (sigversion == SigVersion::BASE || sigversion == SigVersion::WITNESS_V0) return set_error(serror, SCRIPT_ERR_BAD_OPCODE);
+                    valtype& vchScalar = stacktop(-1);
+                    valtype& vchGenerator = stacktop(-2);
+                    valtype& vchRes = stacktop(-3);
+
+                    CPubKey pk(vchGenerator);
+                    CPubKey res(vchRes);
```

Comment:
I think we may want to restrict to compressed keys here. Thoughts?

Reply:
If you disagree, the test should include uncompressed and hybrid keys

Reply:
Agreed. I wanted to restrict to compressed keys, but this is an oversight.